packages/terra-form-input/src/terra-dev-site/doc/common/BlankExample.jsx

```diff
@@ -5,7 +5,7 @@ import Input from '../../../Input';
 const BlankExample = () => (
   <div>
     <Field label="Blank" htmlFor="blank">
-      <Input name="default blank input" id="blank" />
+      <Input name="default blank input" id="blank" ariaLabel="form-input-label" />
```

Comment:
Let's update the ariaLabel value so it matches the label value on the `<Field />` component. e.g.:  `ariaLabel="Blank"`